admin_manual/configuration/server/occ_command.rst

```diff
@@ -985,6 +985,15 @@ info about this group.
    that recently deleted groups from LDAP might be considered as "active" and
    might not be deleted in ownCloud immediately.
 
+**Configuring the LDAP Refresh Attribute Interval**
+
+You can configure the LDAP refresh attribute interval, but not with the ``ldap`` commands. 
+Instead, you need to use the ``config:app:set`` command, as in the following example, which takes a number of seconds to the ``--value`` switch::
+   
+  occ config:app:set user_ldap updateAttributesInterval --value=7200 --type=integer
```

Comment:
[The existing documentation](https://doc.owncloud.org/server/latest/admin_manual/configuration/server/occ_command.html#setting-a-single-configuration-value) indicates that it does.